build.sbt

```diff
@@ -17,15 +17,16 @@ ThisBuild / organization := "za.co.absa"
 
 lazy val scala211 = "2.11.12"
 lazy val scala212 = "2.12.12"
```

Comment:
There is a newer version of Scala 2.12, if you want to use it.
```suggestion
lazy val scala212 = "2.12.18"
```

Reply:
Updated